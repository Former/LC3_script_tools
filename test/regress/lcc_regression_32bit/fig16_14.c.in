90432
5
70123
5
45
0
1
2
5
121222

