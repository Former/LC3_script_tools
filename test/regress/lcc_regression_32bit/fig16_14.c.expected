Enter 10 numbers.
Input number 0 : 90432
Input number 1 : 5
Input number 2 : 70123
Input number 3 : 5
Input number 4 : 45
Input number 5 : 0
Input number 6 : 1
Input number 7 : 2
Input number 8 : 5
Input number 9 : 121222
The input set, in ascending order:
0
1
2
5
5
5
45
70123
90432
121222
